packages/forms-web-app/src/dynamic-forms/options-question.js

```diff
@@ -93,18 +93,20 @@ class OptionsQuestion extends Question {
 				optionData.checked = (',' + answer + ',').includes(',' + optionData.value + ',');
 			}
 
-			// also need to handle dependent fields & set their answers
-			// if (optionData.conditional !== undefined) {
-			// 	let conditionalField = { ...optionData.conditional };
-			// 	conditionalField.fieldName = this.fieldName + '_' + conditionalField.fieldName;
-			// 	conditionalField.value = journey.response.answers[conditionalField.fieldName] || '';
-			// 	optionData.conditional = {
-			// 		html: nunjucks.render(
-			// 			`../views/questions/conditional/${conditionalField.type}.njk`,
-			// 			conditionalField
-			// 		)
-			// 	};
-			// }
+			// handle conditional (dependant) fields & set their answers
+			if (optionData.conditional !== undefined) {
+				let conditionalField = { ...optionData.conditional };
+
+				conditionalField.fieldName = this.fieldName + '_' + option.value + '-value';
```

Comment:
I think using the value as-is in the field name might cause some issues. The value of a form field could be any string which could include characters such as apostrophes, commas and quotation marks which would be invalid in a field name. To use the value, you'd need to sanitise it so it's a valid kebab-case string.